### ISSUE TITLE
Traceback IndexError

### ISSUE THREAD

**Issue by pr0letariat** [chrisallenlane/cheat#329]
```
$ cheat -s pca
Traceback (most recent call last):
  File "/usr/local/bin/cheat", line 57, in <module>
    print(colorize(sheets.search(options['<keyword>'])))
  File "/home/me/.local/lib/python2.7/site-packages/cheat/utils.py", line 23, in colorize
    first_line = sheet_content.splitlines()[0]
IndexError: list index out of range
```

Only had this very recently and only on some searches, not all.

**Comment by pr0letariat** [comment]
Bump... Any ideas?

**Comment by yumitsu** [comment]
Possibly related to #372

**Comment by D1mon** [comment]
same error
`File "/usr/bin/cheat", line 61, in <module>`

**Comment by ipstone** [comment]
I got the same error as well when searching.

**Comment by chrisallenlane** [comment]
Sorry it's taken me so long to address this issue.

This should be resolved as of #382, which should be deployed with a new version of `cheat` in the upcoming days. With that said, I'm going to close this as resolved.

Thanks.
